.github/workflows/actions/build-core-stencil-prerelease/action.yml

```diff
@@ -1,12 +1,12 @@
-name: 'Build Ionic Core with Stencil Prerelease'
-description: 'Build Ionic Core with a Prerelease Build of Stencil'
+name: "Build Ionic Core with Stencil Prerelease"
+description: "Build Ionic Core with a Prerelease Build of Stencil"
 inputs:
   stencil-version:
-    description: 'The NPM tag of @stencil/core to install.'
+    description: "The NPM tag of @stencil/core to install."
     type: string
     required: true
 runs:
-  using: 'composite'
+  using: "composite"
```

Comment:
Can we revert the quotation marks change?

Reply:
Hum some stupid default config from Prettier, already changed it!